### PR TITLE
Claim page

### PR DESCRIPTION
**Pull request by bachstatter**

### CODE REVIEW

File: v2/components/RewardsItem/ClaimRewardsBtn.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    error,
+    gasError,
+    settle,
+    isGasEnabledAndNotFetched,
+    txnHash,
+  } = useClaimRewardsMutation();
+  const handleSubmit = () => {
+    mutate(undefined, {
+      onSuccess: () => {
+        queryClient.refetchQueries(['useRewardsAvailable'], { type: 'active' });
```
Comment (bachstatter) [inline v2/components/RewardsItem/ClaimRewardsBtn.tsx:30]: For some reason this doesn't work 🤔

Comment (bachstatter) [reply v2/components/RewardsItem/ClaimRewardsBtn.tsx:30]: Fixed

File: v2/components/RewardsItem/ClaimLiquidationBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    error,
+    gasError,
+    settle,
+    isGasEnabledAndNotFetched,
+    txnHash,
+  } = useClaimLiquidatorRewardsMutation();
+  const handleSubmit = () => {
+    mutate(undefined, {
+      onSuccess: () => {
+        queryClient.refetchQueries(['useGetLiquidationRewards'], { type: 'active' });
```
Comment (bachstatter) [inline v2/components/RewardsItem/ClaimLiquidationBtn.tsx:24]: for some reason this still doesn't refetch :(

File: v2/components/RewardsItem/RewardsItem.tsx [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+        description={t('staking-v2.earn.staking-rewards.description')}
+        apyReturn={aprData !== undefined ? formatPercent(aprData.toNumber()) : ''}
+        stakedBalance={`${formatNumber(stakedSnx.toNumber()).toString()} SNX`}
+        endDate={feePoolData?.nextFeePeriodStartDate || null}
+        percentCompleted={percentEpochCompleted(
+          feePoolData?.nextFeePeriodStartDate,
+          feePoolData?.feePeriodDuration
+        )}
+        isLoading={isLoading}
+        rewardBalance={`${formatNumber(rewardsData?.snxRewards.toNumber() || 0).toString()} SNX`}
```
Comment (bachstatter) [inline v2/components/RewardsItem/RewardsItem.tsx:334]: This should have sUSD rewards too

Comment (jmzwar) [reply v2/components/RewardsItem/RewardsItem.tsx:334]: It is in the modal though right? I spoke with Jade about it and she thinks it is fine.

File: v2/components/RewardsItem/ClaimLiquidationBtn.tsx (hunk 2 above)
Comment (bachstatter) [reply v2/components/RewardsItem/ClaimLiquidationBtn.tsx:25]: fixed

**Review by jmzwar — APPROVED** [review]

**Pull request merged**